src/testRunner/transpileRunner.ts

```diff
@@ -0,0 +1,132 @@
+import {
+    Baseline,
+    Compiler,
+    getFileBasedTestConfigurations,
+    IO,
+    RunnerBase,
+    TestCaseParser,
+    TestRunnerKind,
+} from "./_namespaces/Harness";
+import * as ts from "./_namespaces/ts";
+import * as vpath from "./_namespaces/vpath";
+
+export class TranspileRunner extends RunnerBase {
+    protected basePath = "tests/cases/transpile";
+    protected testSuiteName: TestRunnerKind = "transpile";
+
+    constructor() {
+        super();
+    }
+

```

Comment:
Can probably just omit this.

```suggestion
```

Reply:
But is there a lint rule for this? 🤔

Reply:
Maybe - it's subtly different from the default constructor though, since I think the real default constructor is closer to:

```js
constructor(...args) {
  super(...args);
}
```